ci/generate_azure_pipelines_matrices.py

```diff
@@ -119,8 +119,11 @@ def __init__(self, platform, build_jobs):
         BuildJob('5.14.0', 'linux', 'desktop', 'gcc_64', 'gcc_64', module='all')
     ]
 )
-mac_build_jobs.append(
-    BuildJob('5.13.2', 'mac', 'desktop', 'clang_64', 'clang_64', module='qcharts qtnetworkauth')
+mac_build_jobs.extend(
+    [
+    BuildJob('5.13.2', 'mac', 'desktop', 'clang_64', 'clang_64', module='qcharts qtnetworkauth'),
+    BuildJob('5.14.2', 'mac', 'desktop', 'clang_64', 'clang_64', module='qcharts qtnetworkauth')
```

Comment:
everywhere qt 5.14.0 is used wrong copy/paste ?

ps: sorry not checked on the qt site which version should be targeted...

Reply:
because #116 report an issue for Qt 5.14.2  on mac.